### PR TITLE
Include identifiers' name and value in output

### PR DESCRIPTION
**Pull request by marwin1991**

### CODE REVIEW

File: secscanner2junit/sast.py [hunk 1]
```diff
@@ -61,6 +61,12 @@ def parse_findings(self, finding, time):
             output += "end line: {end_line}\n".format(end_line=end_line)
         except KeyError:
             pass
+        
+        try:
+            output += "identifiers.name: {identifiers_name}\n".format(identifiers_name=['identifiers'][0]['name'])
```
Comment (angrymeir) [inline secscanner2junit/sast.py:66]: - [ ] `identifiers_name=['identifiers'][0]['name']` should be `identifiers_name=finding['identifiers'][0]['name']`

File: secscanner2junit/sast.py [hunk 2]
```diff
@@ -61,6 +61,12 @@ def parse_findings(self, finding, time):
...
+            output += "identifiers.value: {identifiers_value}\n".format(identifiers_value=['identifiers'][0]['value'])
```
Comment (angrymeir) [inline secscanner2junit/sast.py:67]: - [ ] `identifiers_name=['identifiers'][0]['value']` should be `identifiers_name=finding['identifiers'][0]['value']`

**Review by angrymeir — CHANGES_REQUESTED** [review]

File: secscanner2junit/sast.py (hunk 1 above)
Comment (marwin1991) [reply secscanner2junit/sast.py]: fixed, sorry for stupid error ...

File: secscanner2junit/sast.py (hunk 2 above)
Comment (marwin1991) [reply secscanner2junit/sast.py]: fixed, sorry for stupid error ...

**Review by angrymeir — APPROVED** [review]

**Pull request merged**